editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/internal/l10n/messages.properties

```diff
@@ -70,6 +70,8 @@ autoLayoutActionDescription=Automatic layout of the diagram
 autoLayoutActionTooltip=Does an automatic layout of the diagram
 beanClassExistsValidatorErrorBeanClassMandatory=Bean class name is mandatory.
 beanClassExistsValidatorErrorBeanClassMustExist=Bean class must exist in the Fuse project.
+beanConfigUtilNoMethodsAvailable=No methods available.
```

Comment:
I would precise in our specific case that there is no "public static" method available (might need to  create a different String when there is no method at all)

![image](https://user-images.githubusercontent.com/1105127/28160716-5a77440a-67c1-11e7-9113-9ec40733ccfe.png)

because, i know that i have several methods but should I was not aware that one need to be public static and user can feel that it is buggy although he/she just need more guidance

what do you think?